CHANGELOG.md

```diff
@@ -5,6 +5,48 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.3.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.3.0) - 2023-09-25
+
+This release provides support for the Spend before Sync synchronization
+strategy, and contains the following updates:
+
```

Comment:
This changelog is current as of commit 0309afc0b97c9c159154c04afa7dd75a26091413 on September 26. Before we merge, let's check if there have been any more significant PRs.

Reply:
This suggestion updates the changelog to commit cdddbcc7fe1d114fcfc8ce169bf93fdfd2ac4a67 on September 29: 
https://github.com/ZcashFoundation/zebra/pull/7610#discussion_r1340727295